source/components/recipe-details/style.css

```
@@ -0,0 +1,284 @@
+  margin-left: 1%;
+}
+
+.category-title {
+  text-decoration: underline;
+  font-family: Ubuntu;
+  font-size: 150%;
+  margin-top: 3%;
+  margin-left: 2%;
+}
+
+.category-list {
+  display: flex;
+  margin-left: 5%;
+  font-family: Ubuntu;
+  font-size: 140%;
+  color: #3aad53;
+}
+
+.nutrient-title {
+  text-decoration: underline;
+  font-family: Ubuntu;
+  font-size: 200%;
+  margin-top: 3%;
+  margin-left: 2%;
+}
+
+.nutrient-list {
+  text-align: left;
+  font-family: Ubuntu;
+  font-size: 140%;
+  margin-top: 3%;
+  margin-left: 1%;
+}
+
+.ingredients-box {
+  width: 98.5%;
+  margin-top: 3%;
+  margin-left: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.ingredients-title {
+  text-align: center;
+  text-decoration: underline;
+  font-family: Ubuntu;
+  font-size: 240%;
+  margin-top: 3%;
+}
+
+.ingredients-list {
+  font-family: Ubuntu;
+  font-size: 140%;
+}
+
+li {
+  margin: 3% 0;
+}
+
+.direction-box {
+  width: 98.5%;
+  margin-top: 3%;
+  margin-left: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.direction-title {
+  text-align: center;
+  text-decoration: underline;
+  font-family: Ubuntu;
+  font-size: 200%;
+  margin-top: 3%;
+}
+
+.direction-list {
+  font-family: Ubuntu;
+  font-size: 140%;
+  margin-left: 2%;
+}
+
+.comment-box {
+  width: 98.5%;
+  margin-top: 3%;
+  margin-left: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  flex-flow: row wrap;
+  font-family: Ubuntu;
+  font-size: 140%;
+}
+
+.comment-box-input {
+  display: flex;
+  margin-top: 1%;
+  margin-left: 1%;
+  margin-right: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.user-input {
+  width: 70%;
+  margin-top: 1%;
+  margin-left: 1%;
+  margin-right: 1%;
+  margin-bottom: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  padding: 1%;
+}
+
+.picture-box {
+  margin-top: 1%;
+  margin-left: 3%;
+  margin-right: 1%;
+  margin-bottom: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  padding: 3%;
+}
+
+.submit-box {
+  text-align: center;
+  width: 8%;
+  height: 3%;
+  margin-top: 4%;
+  margin-left: 3%;
+  margin-right: 1%;
+  margin-bottom: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.comment-box-reply {
+  margin-top: 1%;
+  margin-left: 1%;
+  margin-right: 1%;
+  margin-bottom: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.comment-box-reply1 {
+  height: 10vh;
+  margin-top: 1%;
+  margin-left: 1%;
+  margin-right: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  padding: 1%;
+}
+.comment-box-reply2 {
+  height: 10vh;
+  margin-top: 1%;
+  margin-left: 1%;
+  margin-right: 1%;
+  margin-bottom: 1%;
+  border: 1px solid #000000;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  padding: 1%;
+}
```

Please consolidate into one class `.comment-box-reply`. The `margin-bottom` that only appears inside `.comment-box-reply2` can be replaced by adding `padding-bottom` to the container class. See below:
```suggestion
.comment-box-reply {
  height: 10vh;
  margin-top: 1%;
  margin-left: 1%;
  margin-right: 1%;
  border: 1px solid #000000;
  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
  padding: 1%;
}
```